Help/content_server.html

```diff
@@ -104,6 +107,36 @@
          </li>
         </ul>
        </li>
+       <li class="dropdown logged_in_nav logged_out_nav">
+        <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+         <b class="caret">
+         </b>
+         Options
+        </a>
+        <ul class="dropdown-menu">
+         <li>
+          <a class="account_url logged_in_nav" href="#" style="display:none;" target="_blank">
+           <span class="glyphicon glyphicon-new-window">
+           </span>
+           Server Account
```

Comment:
You should build `python build.py --platform=web` for the develop and master branches. The Chrome-specific build goes in chrome-extension-master.